Cargo.toml

```diff
@@ -14,12 +14,13 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 async-trait = "0.1.73"
-axum-core = "0.3"
-base64 = "0.13"
-http = "0.2"
+axum = "0.7"
```

Comment:
I don't think axum is required in top-level deps, the rest of the PR looks great 👍